### PR TITLE
docs: Update new identity documenation in repo

### PR DESCRIPTION
**Pull request by stepansergeevitch**
Update README.md and examples

### CODE REVIEW

File: examples/dbapi.ipynb [hunk 1]
```diff
@@ -36,15 +37,12 @@
    "outputs": [],
    "source": [
     "# Only one of these two parameters should be specified\n",
```
Comment (ptiurin) [inline examples/dbapi.ipynb:39]: This line is no longer relevant and confusing now.

**Review by ptiurin — APPROVED** [review]
Apart from that LGTM

**Comment by ptiurin** [comment]
Need to hold off merging this until we change https://docs.firebolt.io/developing-with-firebolt/connecting-with-Python.html
Otherwise, all users will be directed to the updated docs

**Comment by stepansergeevitch** [comment]
Merging since the driver documentation version has been locked to `0.x` in docs

**Pull request merged**